app/models/subscription.rb

```diff
@@ -9,9 +9,92 @@
 #  token                :string
 #  created_at           :datetime         not null
 #  updated_at           :datetime         not null
+#  canceled_at          :datetime
```

Comment:
Needed this for the `devise_token_auth` gem we're using. There's some hacky things I'm doing to get it to work. We may go with a different solution later on, but for now, this is fine.